packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -7825,19 +7825,19 @@ export class LayerBase extends React.Component<ILayerProps, ILayerBaseState> {
     // (undocumented)
     componentDidUpdate(): void;
     // (undocumented)
-    componentWillMount(): void;
-    // (undocumented)
     componentWillUnmount(): void;
     // (undocumented)
-    componentWillUpdate(): void;
-    // (undocumented)
     static defaultProps: ILayerProps;
     // (undocumented)
     render(): React.ReactNode;
-    }
+    // (undocumented)
+    UNSAFE_componentWillMount(): void;
+    // (undocumented)
+    UNSAFE_componentWillUpdate(): void;

```

Comment:
@ecraig12345 @jdhuntington I made this a draft PR because I'm still pondering whether we should consider this an API breaking change. The public API changes, but will end user overrides actually break?

I did a [simple CodePen](https://codepen.io/jasongore/pen/BaBjrXg) that shows that even when both versions of the function are named, both will still get called. Existing user implementations should continue to work. 

The only exception I can think of is if a user intended to completely override and disable a component's lifecycle method. In these types of cases, this PR would break those cases. But is that a practical concern?

Reply:
I don't think this should be considered a breaking API change. These are framework methods, not things that consumers (should) take a dependency on.